package.json

```diff
@@ -26,8 +26,27 @@
     "test": "mocha $MOCHA_FLAGS"
   },
   "files": [
-    "lib/",
-    "src/",
```

Comment:
create a different folder if we have something else that we don't want to distribute...